src/main/scala/mesosphere/util/monitor/HeartbeatMonitor.scala

```diff
@@ -0,0 +1,81 @@
+package mesosphere.util.monitor
+
+import javax.inject.Inject
+
+import akka.actor._
+import akka.event.LoggingReceive
+import scala.concurrent.duration._
+
+// HeartbeatActor monitors the heartbeat of some process and executes handlers for various conditions.
```

Comment:
ack. it's designed with scheduler heartbeat req's in mind. but there's no scheduler specific glue in here ATM. we'll want some kind of heartbeat monitoring once we switch to the v1 API as well. the bits that integrate with the mesos scheduler may look much different. these more generic monitor bits (as they currently stand) probably won't. which, to me, is an argument to keep things separate.

moving to a separate package is fine. i really don't like util either. i cry a little bit inside thinking about all the boilerplate/overhead required to use the service/actor/impl/etc/etc "core" packaging structure that we've established for bigger modules -- I only want a simple actor after all.